### DISCUSSION TITLE
Keypad Motion Event...please

### DISCUSSION THREAD

**Discussion by bthome** [tsightler/ring-mqtt#333] [Feature Requests]
Is it possible to have the Ring Keypad v2 embedded motion sensor event captured and published.  That would provide extra motion sensors when close to the keypad.  Thanks.

**Comment by tsightler** [comment]
As far as I can tell those events are not sent via the API.  I believe it's just a proximity sensor which is completely handled locally within the keypad itself.  If you can demonstrate that this data is somehow available in the API, then it could be considered.

**Comment by GizmoGoody** [comment]
Vote Here: https://community.ring.com/t/keypad-motion-detector/30383

**Comment by tsightler** [comment]
Duplicate of https://github.com/tsightler/ring-mqtt/discussions/526
